website_sale_checkout_skip_payment/tests/test_checkout_skip_payment.py

```diff
@@ -11,9 +11,6 @@ def setUp(self):
         self.partner.with_context({"res_partner_search_mode": "customer"}).write(
             {"skip_website_checkout_payment": True}
         )
-        # Delete optional products for avoid popup window
-        product = self.env.ref("product.product_product_4_product_template")
```

Comment:
Hello, what is the reason to remove these test lines?

Reply:
I removed them because `optional_product_ids` is no longer a field `product.template`. That's why I think those lines are not needed anymore.